package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "@mapbox/mvt-fixtures": "^3.10.0"
+  }
```

Comment:
@ebrelsford did you intend to add this at the root of the project or within `parser/js/package.json` instead?

Reply:
Oof was re-playing some changes and messed this one up, thanks.